pkg/edit/vars.go

```diff
@@ -56,6 +55,34 @@ func initVarsAPI(ed *Editor, nb eval.NsBuilder) {
 // ~> echo $b:foo
 // foo
 // ```
+//
+// Because of the above behavior this mechanism is perhaps most useful for adding a module namespace
```

Comment:
Good point. I would argue it's prescriptive since I don't believe anyone has identified a use case for creating a non-namespace, non-function, variable in the REPL using this mechanism. Hence my earlier comments on IRC that perhaps this command should only allow such objects by default; possibly providing an option (tongue-in-cheek: `&i-know-what-i-am-doing`) to define a REPL var with some other value (scalar, list, map).